user-guide/Cloud_Platform/Catalog/Register_Catalog_Item.md

```diff
@@ -4,19 +4,28 @@ uid: Register_Catalog_Item
 
 # Registering a Catalog item
```

Comment:
Should we not just remove all references to the UI registration since we are going to remove that anyway?

Reply:
i wasn't aware it was going to go away :) . better to delete it in that case indeed